pom.xml

```diff
@@ -414,11 +416,12 @@
                     </execution>
                 </executions>
                 <configuration>
-                    <repository>docker.io/spanconnector/kafka-spanner-connector</repository>
+                    <repository>docker.io/${docker.repository.name}</repository>
```

Comment:
I think there should be `docker.repository.name` operty with default value. Also `docker.io` should not be hardocded but be part of the repo name.

Reply:
Hi Jiri! Thanks for the suggestion!

Do you have any ideas on what could be the default value for the docker repository name? The one I used to use is a private internal-only repository.

Reply:
It could be the one you mentioned as push is disabled by default. Or you can use `debezium` namespace like `debezium/kafka-spanner-connector`